.github/actions/commit-via-pr/action.yml

```
@@ -80,12 +80,20 @@ runs:
+        if ! gh pr create \
           --base main \
           --head "$BRANCH" \
           --title "$COMMIT_MESSAGE" \
-          --body "$PR_BODY"
+          --body "$PR_BODY"; then
+          echo "::error::Failed to create PR for $BRANCH"
+          exit 1
+        fi
 
         gh pr merge "$BRANCH" --auto --squash --delete-branch
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)